sswlib/src/main/java/components/CombatVehicle.java

```diff
@@ -389,13 +389,12 @@ public int GetCurrentBV() {
 
     public double GetTotalCost() {
         // final cost calculations
-        // 8/7/2012 Commented out the GetAmmoCosts addition to match the way Mech works - GKB
-        return (( GetChassisCost() + GetEquipCost() ) * GetCostMult() * GetConfigMultiplier()); //+ GetAmmoCosts();
+        return ((GetChassisCost() + GetEquipCost() ) * GetCostMult() * GetConfigMultiplier()) + GetAmmoCosts();
     }
     
     public double GetDryCost() {
         // returns the total cost of the mech without ammunition
-        return ( GetEquipCost() + GetChassisCost() ) * GetCostMult() * GetConfigMultiplier();
+        return (GetEquipCost() + GetChassisCost()) * GetCostMult() * GetConfigMultiplier();
     }

```

Comment:
Next time this area is touched, we should just have `GetTotalCost` reference `GetDryCost` to minimize Code Duplication

Reply:
I can update these when I add VTOL.